vnext/Shared/WinRTWebSocketResource.cpp

```diff
@@ -87,11 +87,22 @@ string HResultToString(hresult &&result) {
 
 namespace Microsoft::React {
 
+// private
+WinRTWebSocketResource::WinRTWebSocketResource(
+    IMessageWebSocket &&socket,
+    Uri &&uri,
+    vector<ChainValidationResult> certExeptions) noexcept
```

Comment:
this should be && in order to enable to move it

Reply:
Good point. Will submit that change to master and add to the backports before merging.